sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -183,7 +183,10 @@ private synchronized void instanceSetSenderId(String senderId) {
     }
 
     /**
-     * @deprecated
+     * @deprecated Future releases will not alter the state of Firebase Analytics data collection by
```

Comment:
Please commit (not push but at least commit) using Android Studio and enable this:

<img width="278" alt="screen shot 2017-11-17 at 2 17 31 pm" src="https://user-images.githubusercontent.com/623672/32971442-16844ffa-cba2-11e7-9481-4386d01f3bdc.png">

As for this particular block, please format the file again.

Reply:
What should the format be?